tests/disk.rs

```diff
@@ -1,34 +1,149 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
+#[cfg(all(feature = "system", feature = "disk"))]
+fn should_skip() -> bool {
+    if !sysinfo::IS_SUPPORTED_SYSTEM {
+        return true;
+    }
+
+    let s = sysinfo::System::new_all();
+    if s.physical_core_count().unwrap_or_default() == 0 {
+        return true;
+    }
+    false
```

Comment:
```suggestion
    s.hysical_core_count().unwrap_or_default() == 0
```

Reply:
Done (and brought back a comment that I should have kept when factoring this logic out of the existing tests.)